src/brogue/Monsters.c

```diff
@@ -3891,8 +3964,9 @@ void demoteMonsterFromLeadership(creature *monst) {
     for (int level = 0; level <= DEEPEST_LEVEL; level++) {
         if (!BROGUE_VERSION_ATLEAST(1,9,1) && level > 0) break;
         // we'll work on this level's monsters first, so that the new leader is preferably on the same level
-        creature *firstMonster = (level == 0 ? monsters->nextCreature : levels[level-1].monsters);
-        for (follower = firstMonster; follower != NULL; follower = follower->nextCreature) {
+        creatureList *nearbyList = level == 0 ? &monsters : &levels[level-1].monsters;
```

Comment:
Could you keep the parens?

```suggestion
        creatureList *nearbyList = (level == 0 ? &monsters : &levels[level-1].monsters);
```

Reply:
There's another instance just after this